build-support/docker/Build-UI.dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG ALPINE_VERSION=3.13
 FROM alpine:${ALPINE_VERSION}
 
-ARG NODEJS_VERSION=14.16.1-r1
+ARG NODEJS_VERSION=14.17.6-r0
```

Comment:
I was having issues running `make ui` locally and the error messages hinted that I should bump the version. But now that I think about it I'm wondering how CI was building it fine 🤔

Reply:
Hey @kisunji 

First thing I noticed here was related to this change, I'm not sure if things have changed upstream since you made this change, but I had to move the `npm` installation to use a different name/version also in order to get `make ui` working. I was gonna PR this separately, but now not so sure. Maybe we should bump to alpine 3.14 as a completely different PR (are we gonna need that bump on 1.10.x/1.9.x branch etc?), kinda up to you.

Reply:
@johncowen if you can make a separate PR for the version bumps (including the alphine) it would be appreciated! My PR isn't dependent on these changes at all; I just needed this version to build locally.